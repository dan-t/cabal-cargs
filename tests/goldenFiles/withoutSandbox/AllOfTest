-itests -W -optP-DTEST  -idist/build/autogen -Idist/build/autogen -optP-include -optPcabal_macros.h