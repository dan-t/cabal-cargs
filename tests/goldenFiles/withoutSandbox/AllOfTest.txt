-itests -W -idist/build/autogen -Idist/build/autogen -optP-include -optPcabal_macros.h